### PR TITLE
Completed TODO: Resolve importing errors and use importing format as above.

### PR DESCRIPTION
**Pull request by Valerievich**
All the tests were passed. Local build works well.

### CODE REVIEW

File: mimesis/providers/generic.py [hunk 1]
```diff
@@ -1,20 +1,10 @@
...
-from .internet import Internet
-from .numbers import Numbers
-from .path import Path
-from .personal import Personal
-from .science import Science
-from .text import Text
-from .transport import Transport
-from .units import UnitSystem
-from .cryptographic import Cryptographic
+                               Code, Datetime, Development, File, Food, Games,
```
Comment (sobolevn) [inline mimesis/providers/generic.py]: Could you please follow "one-line-one-import" standard?

Comment (Valerievich) [inline mimesis/providers/generic.py]: Yes, I could. But current version of generic.py already breaks the rule, doesn't it? Should I refactor all lines with imports? I suppose @lk-geimfari wanted to use more compact view in this case.

Comment (lk-geimfari) [inline mimesis/providers/generic.py]: @sobolevn It seemed to me that this way is more compact, but if you think that this is not the better way of importing, then we can request refactoring.

P.S Yes, this way was really requested by me. Thanks @Valerievich

Comment (sobolevn) [inline mimesis/providers/generic.py]: I'm totally fine with both versions

**Pull request merged**